### ISSUE TITLE
Accessibility: Tables with unique accessible names

### ISSUE THREAD

**Issue by BinquanWang-USDA** [tripal/tripal#2207]
### This feature attempts to solve an existing problem

No

### I am open to developing or collaborating on an extension module if this is not a good fit for Tripal Core

None

### Is this feature urgent?

Yes

### Description

Various tables generated by Tripal, for instance the "Summary" section on an Organism page, doesn't have their unique accessible names. For the sake of accessibility, (screen readers, etc.), enabling the ability to add unique accessible names to the tables is required. 
Please see: [WCAG21 1.3.1/2.4.6 (A/AA)] Unique accessible name required:
Data tables must have unique accessible names to help identify them and to differentiate between each table on the page. The accessible name is provided by 'aria-labelledby' > 'aria-label' > caption > 'title' > 'summary'. If two tables contain the same information then number them in the accessible name.

### Your specific use case

It is required as per accessibility standards.

### Generally Applicable

All Tripal sites should take accessibility standards into account (see below).

### Additional information/screenshots

Accessibility standards are important to adhere to and make use of, and not just for passing tests. Through recent scans of a Tripal site that is currently in development, there are a number of issues that may fall to Tripal core to implement. Therefore, I also suggest a new label for Tripal issues be created - "accessibility". Most of these will also be "enhancements", an existing label, but it will be useful to highlight these.

**Comment by laceysanderson** [comment]
Thanks for finding this @Ferrisx4. I've added the label as requested and marked this as high priority since it could render the site unusable for those using accessibility devices such as screen readers.

**Comment by Ferrisx4** [comment]
The particular elements described in this issue are specific to this particular website and not impacted by Tripal, but instead created with Field Groups module. The solution will be to apply these required elements through theming and templating instead.

That being said, Tripal is open to making its elements and content more accessible so any new issues related to this will be considered high priority.
